hack/test-gce/nodeagent_install.sh

```diff
@@ -0,0 +1,282 @@
+#! /bin/bash
+
+set -e
+
+#Golang version
+GO_VERSION=${GO_VERSION:-"1.18.7"}
+
+pushd $HOME
+echo -e "## SETTING UP THE HOSTNAME NODEAGENT-A\n"
+sudo hostnamectl set-hostname nodeagent-a
```

Comment:
why do we set host name, and it set same name for all node agents?

Reply:
I comment these two line